### PR TITLE
Add method/classes to get a user's tagged places.

### PR DESCRIPTION
**Pull request by xsorifc28**
I haven't been able to test this yet, but it should work?

### CODE REVIEW

File: simple-fb/src/main/java/com/sromku/simple/fb/entities/PlaceTag.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        @SerializedName(LOCATION)
+        private Location mLocation;
+
+        public Location getLocation() {
+            return mLocation;
+        }
+    }
+
+
+    public class Location {
```
Comment (sromku) [inline simple-fb/src/main/java/com/sromku/simple/fb/entities/PlaceTag.java]: @xsorifc28 Please change the existing `Location` class and copy this inner class into existing one

File: simple-fb/src/main/java/com/sromku/simple/fb/entities/PlaceTag.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    public String getCreatedTime() {
+        return mCreatedTime;
+    }
+
+    public String getId() {
+        return mId;
+    }
+
+
+    public class Place extends IdName{
```
Comment (sromku) [inline simple-fb/src/main/java/com/sromku/simple/fb/entities/PlaceTag.java]: @xsorifc28 Please change the existing `Place` class and copy this inner class into existing one

**Comment by sromku** [comment]
@xsorifc28 The pull request is awesome and in fact you also fixed the bug of `Place` and `Location` entities. The library already contains both objects in *com.sromku.simple.fb.entities* package. 
Please replace current [`Place`](https://github.com/sromku/android-simple-facebook/blob/master/simple-fb%2Fsrc%2Fmain%2Fjava%2Fcom%2Fsromku%2Fsimple%2Ffb%2Fentities%2FPlace.java) and [`Location`](https://github.com/sromku/android-simple-facebook/blob/master/simple-fb%2Fsrc%2Fmain%2Fjava%2Fcom%2Fsromku%2Fsimple%2Ffb%2Fentities%2FLocation.java) classes in the lib with your implementation of these classes. And then, delete your inner classes. 
Let me know if you can handle this or have any issues with this change. Thanks.

**Comment by xsorifc28** [comment]
I wasn't sure if facebook was using `Place` and `Location` differently so I made two inner classes. Since you suggest to merge them into the existing class, I assume this will not break any other `get` methods.

Will commit the changes soon!

**Comment by xsorifc28** [comment]
@sromku Updated!

**Comment by xsorifc28** [comment]
I am getting this error when building, so I cannot test:

Error:Gradle: Execution failed for task ':simple-fb:processDebugAndroidTestManifest'.
> java.lang.RuntimeException: Manifest merger failed : uses-sdk:minSdkVersion 9 cannot be smaller than version 15 declared in library C:\Users\Samed\AndroidStudioProjects\android-simple-facebook\simple-fb\build\intermediates\exploded-aar\com.facebook.android\facebook-android-sdk\4.6.0\AndroidManifest.xml
	Suggestion: use tools:overrideLibrary="com.facebook" to force usage

The way I currently test is by importing to another project via gradle. Manually adding the library also did not work. Any suggestions on how to fix this? Should I just use studio's `overrideLibrrary` suggestion?

**Comment by sromku** [comment]
@xsorifc28 much thanks! 
The issue with facebook is that they changed their lib to be supported from min SDK 15. And they also updated gradle version to 1.2.3 which fails builds when your app has lower min SDK version. 

There is a solution for this by using  `overrideLibrary` as you told. This should be added to your manifest if you want to use lower sdk:
``` xml
<uses-sdk android:minSdkVersion="9"
          tools:overrideLibrary="com.facebook.android"/>
```
So it looks that to prevent such issue, I will update this lib to use min SDK 15 as facebook did and for those who want to use lower version, will have to add `tools:overrideLibrary` option in their project.

**Comment by xsorifc28** [comment]
Great, thanks!

**Comment by sromku** [comment]
@xsorifc28 merged, tested. I added new sample screen to the sample app menu with your new tagged places option. It works great!
Also new wiki page: https://github.com/sromku/android-simple-facebook/wiki/Get-tagged-places

new version `4.1.0` is released with your new api:
```
 dependencies {
    compile 'com.sromku:simple-fb:4.1.0'
}
```

Much thanks! Roman

**Comment by xsorifc28** [comment]
Thank you for the update! This should also fix an issue I was having with hometown page 'Location'

You're awesome!

**Comment by sromku** [comment]
You too! :smiley:

**Pull request merged**